### PR TITLE
feat: add labels to bottom tab bar

### PR DESCRIPTION
**Pull request by BhuvaneshPatil**
### Details

New feature -  add labels to icons on bottom tab bar

### Fixed Issues

$ https://github.com/Expensify/App/issues/47339
PROPOSAL: https://github.com/Expensify/App/issues/47339#issuecomment-2286559311

### Tests

1. Verify if non-active icon has textSupporting color and non bold label
2. Selected icon has text color and bold label
3. Verify if 1 and 2 works with emoji status set
4. Verify if 1 and 2 works with copilot set 

- [x] Verify that no errors appear in the JS console

### Offline tests

Same as above

### QA Steps

Same as above
- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/user-attachments/assets/fe668002-9809-4877-895b-332b8f4a83d3

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/user-attachments/assets/1063f2ce-5b36-480d-8f82-0331dc8a71c2

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/user-attachments/assets/d0f4ec98-8ff0-4144-9d75-77499ba17a32

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/user-attachments/assets/22614a9d-2267-41ea-a89d-5d9b2240f963

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/user-attachments/assets/996da30e-45a7-4ada-ad05-655c9384f336

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/user-attachments/assets/190928d2-9a05-44ad-bc79-8b5eace2615b

</details>

### CODE REVIEW

**Comment by BhuvaneshPatil** [comment]
Android Browser - 


Screenshot 2024-08-26 at 7 30 27 PM


Screenshot 2024-08-26 at 7 30 21 PM


Screenshot 2024-08-26 at 7 30 05 PM


Screenshot 2024-08-26 at 7 29 52 PM

**Comment by BhuvaneshPatil** [comment]
android native-
Screenshot 2024-08-26 at 7 21 00 PM
Screenshot 2024-08-26 at 7 20 51 PM
Screenshot 2024-08-26 at 7 20 24 PM
Screenshot 2024-08-26 at 7 20 16 PM
Screenshot 2024-08-19 at 7 10 01 PM

**Comment by BhuvaneshPatil** [comment]
IOS Native -
Screenshot 2024-08-25 at 9 58 12 PM
Screenshot 2024-08-25 at 9 57 55 PM
Screenshot 2024-08-25 at 9 57 34 PM
Screenshot 2024-08-25 at 9 56 40 PM

**Comment by BhuvaneshPatil** [comment]
Screenshot 2024-08-19 at 7 10 18 PM
Screenshot 2024-08-19 at 7 10 01 PM

**Comment by rushatgabhane** [comment]
Makes sense

**Comment by rushatgabhane** [comment]
@Expensify/design 

We need your feedback on the design - https://github.com/Expensify/App/pull/48003#issuecomment-2310329673

**Comment by shawnborton** [comment]
So we need to adjust the gap between icon + label so that all of the labels line up perfectly. See the big difference here?
CleanShot 2024-08-27 at 10 59 33@2x

The way to handle this is to use different gaps for the first two tabs vs the settings tab. I have it so that the first two tabs are using 6px gap, and then the Settings tab uses 2px gap:

https://github.com/user-attachments/assets/459fdc26-ce4d-4816-bcad-dfd5b56dbd19

I also tried to reduce the 2px gap between the green ring and the avatar to be just 1px so we save some space there too. cc @Expensify/design for thoughts on all of this

**Comment by shawnborton** [comment]
Also, the tabs that are not active should use textSupporting for the color, not our standard text color. 
CleanShot 2024-08-27 at 11 05 23@2x

**Comment by dannymcclain** [comment]
Love all those comments Shawn! Something looks weird with the 1px gap around the selected avatar though... It doesn't look even to me? (Closer on the sides than it is on the top... No idea why that would be)
CleanShot 2024-08-27 at 10 17 37@2x

**Comment by shawnborton** [comment]
Oh boy it was set to 33x34... good eye, my b! This is an updated version that looks even:
CleanShot 2024-08-27 at 11 21 14@2x

**Comment by rushatgabhane** [comment]
@BhuvaneshPatil bump on the above

**Comment by BhuvaneshPatil** [comment]
@rushatgabhane I modified the code for given margins that is - 6px for inbox and search and 2px for avatar. It looks good when we are on inbox or search page but when we have green ring it pushes the label down (for Settings)

**Comment by shawnborton** [comment]
>  It looks good when we are on inbox or search page but when we have green ring it pushes the label down (for Settings)

Is there a way to absolutely position the green ring? That's what We have in Figma I believe.

**Comment by BhuvaneshPatil** [comment]
Screenshot 2024-09-07 at 11 55 55 PM
<br/>
Screenshot 2024-09-07 at 11 55 23 PM
<br/>
Screenshot 2024-09-07 at 11 54 53 PM
<br/>
Screenshot 2024-09-07 at 11 54 12 PM

**Comment by shawnborton** [comment]
Looks like the alignment is still slightly off and incorrect. Can you keep adjusting please? Happy to help provide any direction if needed, just let us know. Going to spin up a test build in the meantime.

**Comment by shawnborton** [comment]
Hmm the iOS app crashes when I try to open it from the test build. Any ideas?

**Comment by BhuvaneshPatil** [comment]
I am able to run the app locally on the same branch.

**Comment by dubielzyk-expensify** [comment]
There's some funky jumps happening:
CleanShot 2024-09-09 at 13 30 30

Also, do we want the inactive label text color to be secondary color or primary @shawnborton ?

**Comment by shawnborton** [comment]
Good catch, we should not see any "jumping" besides the label text becoming bold. 

And the inactive labels should use textSupporting color as you mentioned.

**Comment by shawnborton** [comment]
@BhuvaneshPatil can you try running the test build on iOS from above (if you have an iPhone)? It's still crashing for me.

**Comment by BhuvaneshPatil** [comment]
The jumpy behaviour is because I am adjusting top margins when we have ring or emoji status,I have done for keeping the text on same line

**Comment by shawnborton** [comment]
We need to remove the jumpy behavior and make sure everything feels "fixed" in place - can you please adjust?

**Comment by BhuvaneshPatil** [comment]
@shawnborton I made the border absolute, it will solve the issue for selected avatar, can you please provide design/gap numbers of margin for avatar with emoji status please

**Comment by shawnborton** [comment]
@BhuvaneshPatil the emoji status should also be absolutely positioned and not change anything about how the avatar icon is placed. 

Is this ready for a new test build btw? You said you made the ring absolutely positioned but I didn't see any new commits come through.

**Comment by shawnborton** [comment]
I just tried the most recent build and I am still getting the jumpiness when selecting the Settings tab.

**Comment by BhuvaneshPatil** [comment]
I have changes locally, I was thinking of making all changes first then verify. Does that work?

**Comment by shawnborton** [comment]
Sounds good. Please push your latest changes when you can so we can test from there. Please remember to adjust the font color like we mentioned above as well, thanks!

**Comment by BhuvaneshPatil** [comment]
@shawnborton can you please provide a design like you've shared [here](https://github.com/Expensify/App/pull/48003#issuecomment-2312815088). For the case where we have emoji status and labels

https://github.com/user-attachments/assets/c3dd3138-bade-474a-9d6a-98b4f6b3cbef

Screenshot 2024-09-10 at 8 11 13 AM

**Comment by dubielzyk-expensify** [comment]
I think we'll have to put the status bubble in the top right corner with this new change or add more spacing between the icon and the label. Keen to hear what @shawnborton thinks.

**Comment by BhuvaneshPatil** [comment]
Issue with placing emoji status on top right corner is that we show tiny green dot if user has some action required to perform.

For example when I created new account and email is not verified we show tiny green dot.

**Comment by dubielzyk-expensify** [comment]
Ah of course. My bad for forgetting. Let's see what Shawn says. We'll find a solution either way 👍

**Comment by dubielzyk-expensify** [comment]
> but I can totally see moving it up and to the left by like 1-2 pixels to make a little more room for the label.

Definitely like that thinking. Tested it with inspect mode and changing it from `-4px` to `-2px` works well.

**Comment by shawnborton** [comment]
Okay here's how I think we can approach this.

First, the wrapper that wraps the avatar portion needs to be 28x28. Then let's give it position relative and move it to `top: -2px` so we vertically pull it up a bit. This gives us a little extra space below but optically it looks balanced.
CleanShot 2024-09-11 at 13 46 59@2x

For the emoji portion, I think that can be `-6px` from the right and bottom:
CleanShot 2024-09-11 at 13 48 59@2x

Can you try that and see how it goes?

**Comment by shawnborton** [comment]
We're so close! Can you give the wrapper that holds the Settings icon a style of `top: -2px` so we pull it up slightly to create a bit more separation between the icon and label? I think it still maintains optical alignment even though we're pulling it up slightly:
CleanShot 2024-09-12 at 10 13 17@2x

**Comment by shawnborton** [comment]
Hmm no idea why iOS failed, maybe you need to pull main?

**Comment by shawnborton** [comment]
Otherwise this is feeling pretty good:

https://github.com/user-attachments/assets/897414f4-c40e-4401-b432-14f04f9ca56d


cc @Expensify/design to take it for a spin too!

**Comment by dannymcclain** [comment]
I was just looking at it on web—overall feels pretty good to me too!

It gets a little intense/tight when you also have an emoji status saved, but that's not really this project's fault. I feel like we can noodle on that for later.

CleanShot 2024-09-12 at 08 26 33@2x

**Comment by shawnborton** [comment]
Agree - it's definitely tight but I think we finally got it to an acceptable spot at least! Triggered a new build btw.

**Comment by shawnborton** [comment]
Hmm no idea why this won't build for iOS. 

Either way, I'm definitely curious for @dubielzyk-expensify to get a chance to weigh in on the styles too, but otherwise I think @rushatgabhane you can start reviewing this.

Also cc @anmurali in case you want to take this for a spin as well.

**Comment by BhuvaneshPatil** [comment]
I am trying to turn developer mode on in my iphone but I don't see the option (I am on ios 18).

**Comment by dubielzyk-expensify** [comment]
On web it's looking great!

On Android the icons shift a tad bit on my Pixel 7. Is it possible to avoid this?

https://github.com/user-attachments/assets/9df5ca6d-1809-4c87-af35-cce82235b386

**Comment by shawnborton** [comment]
Interesting catch Jon. I wonder if we need to set a fixed height for the labels? Which would be `14` technically since I think that is the desired line height we have for the micro font size.

**Comment by BhuvaneshPatil** [comment]
https://github.com/user-attachments/assets/49c5d87e-bb37-4cd8-97d6-d468e0976d90

**Comment by shawnborton** [comment]
@dubielzyk-expensify @BhuvaneshPatil did those latest line height changes fix the issue Jon reported? I spun up a new test build so we can check again.

**Comment by shawnborton** [comment]
In your video, it looks like the slight jump/movement is still happening. Any ideas how to fix that?

**Comment by rushatgabhane** [comment]
@ design - when accessing as a copilot, this is how it looks. Let me know if it looks okay

image

**Comment by dubielzyk-expensify** [comment]
> @ design - when accessing as a copilot, this is how it looks. Let me know if it looks okay

Good catch. I think this should have the same treatment as we've made to the status emoji above cause that doesn't look great at the moment :) Keen to hear if @Expensify/design has thoughts though.

**Comment by shawnborton** [comment]
> when accessing as a copilot, this is how it looks. Let me know if it looks okay

Oh interesting, I could have sworn we decided not to do that in the end? If we are planning to keep that, let's just occupy the same space as the status I suppose. cc @dylanexpensify in case you have thoughts too

**Comment by BhuvaneshPatil** [comment]
I have applied same styles to co-pilot avatar that appears on the place of emoji status.

Also I have given height of 16px. The root cause of moving icon hen clicked is when text gets bold it changes its height and width and that causes layout to shift. Please verify attached video looks fine.

https://github.com/user-attachments/assets/e83e7275-5040-4ab2-a4a0-8ef97b418221

**Comment by shawnborton** [comment]
How are you getting this particular avatar to show up as the Copilot's avatar? It looks broken:
CleanShot 2024-09-16 at 07 54 17@2x

Size-wise the layout looks okay though, it's just the specific avatar being used that is throwing me off.

**Comment by BhuvaneshPatil** [comment]
I intentionally added some random email (for testing)

**Comment by shawnborton** [comment]
Sure, but is that the fallback avatar being used? It feels incorrect to me.

**Comment by BhuvaneshPatil** [comment]
By looking at code - https://github.com/Expensify/App/blob/main/assets/images/avatars/fallback-avatar.svg
Is the fallback avatar that's being used.

**Comment by shawnborton** [comment]
Ah but notice how much negative space is around the fallback avatar:
CleanShot 2024-09-16 at 10 52 16@2x

That same space around the shape should be maintained here as well.

**Comment by BhuvaneshPatil** [comment]
I am not sure if I am getting what you are trying to say.
By looking in browser, we have given 28x28 dimensions for that. So maybe cause of that it looks tighter

**Comment by shawnborton** [comment]
Something looks off though, even if we scale it down to 28x28, it should retain the slight amount of negative space around the icon. So let's take a quick look into that but I also don't want to block progress so we can get this merged.

Can you share a screenshot of what it looks like when the Copilot avatar is available for comparison to the fallback version?

**Comment by BhuvaneshPatil** [comment]
sure

mac web - 
Screenshot 2024-09-17 at 11 22 04 AM

IOS native-  
Screenshot 2024-09-17 at 11 22 19 AM

**Comment by shawnborton** [comment]
Okay great - see the difference? The iOS version is not the same as web. Web looks correct though:
CleanShot 2024-09-17 at 09 56 55@2x

**Comment by BhuvaneshPatil** [comment]
Right, I will try to fix that in this PR itself

**Comment by BhuvaneshPatil** [comment]
android_native
ios_native
iphone_web
android_web

**Comment by shawnborton** [comment]
That seems better. I think we can start to get this into final review! cc @rushatgabhane

**Comment by rushatgabhane** [comment]
@BhuvaneshPatil are you sure you pushed commits for the above?

**Comment by BhuvaneshPatil** [comment]
not yet, wanted to get confirmation first

**Comment by BhuvaneshPatil** [comment]
@rushatgabhane Completed checklist

**Comment by rushatgabhane** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [ ] I verified that all the inputs inside a form are aligned with each other.
    - [ ] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>

**Comment by BhuvaneshPatil** [comment]
Quick bump @rushatgabhane

File: src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/BottomTabBar.tsx [hunk 1]
```diff
@@ -155,6 +156,12 @@ function BottomTabBar({selectedTab}: BottomTabBarProps) {
                             <View style={styles.bottomTabStatusIndicator(chatTabBrickRoad === CONST.BRICK_ROAD_INDICATOR_STATUS.INFO ? theme.iconSuccessFill : theme.danger)} />
                         )}
                     </View>
+                    <Text
+                        textAlign="center"
```
Comment (rushatgabhane) [inline src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/BottomTabBar.tsx:160]: please use `style={styles.textAlignCenter}`

File: src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/BottomTabBar.tsx [hunk 2]
```diff
@@ -173,6 +180,12 @@ function BottomTabBar({selectedTab}: BottomTabBarProps) {
                             height={variables.iconBottomBar}
                         />
                     </View>
+                    <Text
+                        textAlign="center"
```
Comment (rushatgabhane) [inline src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/BottomTabBar.tsx:184]: same here `styles.textAlignCenter`

File: src/pages/home/sidebar/BottomTabAvatar.tsx [hunk 3]
```diff
@@ -66,9 +74,15 @@ function BottomTabAvatar({isCreateMenuOpen = false, isSelected = false}: BottomT
                 role={CONST.ROLE.BUTTON}
                 accessibilityLabel={translate('sidebarScreen.buttonMySettings')}
                 wrapperStyle={styles.flex1}
-                style={styles.bottomTabBarItem}
+                style={[styles.bottomTabBarItem]}
             >
                 {children}
+                <Text
+                    textAlign="center"
```
Comment (rushatgabhane) [inline src/pages/home/sidebar/BottomTabAvatar.tsx:81]: same here `styles.textAlignCenter`

**Review by rushatgabhane — CHANGES_REQUESTED** [review]
Small change. Rest LGTM

**Comment by BhuvaneshPatil** [comment]
@rushatgabhane Updated the PR

**Review by rushatgabhane — APPROVED** [review]

**Review by marcochavezf — APPROVED** [review]
LGTM

**Pull request merged**